packages/quick_actions/quick_actions_ios/ios/Classes/QuickActionsPlugin.swift

```diff
@@ -0,0 +1,90 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+
+public final class QuickActionsPlugin: NSObject, FlutterPlugin {
+
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(
+      name: "plugins.flutter.io/quick_actions_ios",
+      binaryMessenger: registrar.messenger())
+    let instance = QuickActionsPlugin(channel: channel)
+    registrar.addMethodCallDelegate(instance, channel: channel)
+    registrar.addApplicationDelegate(instance)
+  }
+
+  private let channel: FlutterMethodChannel
+  private let shortcutStateManager: FLTShortcutStateManager
+  /// The type of the shortcut item selected when launching the app.
+  private var launchingShortcutType: String? = nil
+
+  // TODO: (hellohuanlin) remove `@objc` attribute and make it non-public after migrating tests to Swift.
+  @objc
+  public init(
+    channel: FlutterMethodChannel,
+    shortcutStateManager: FLTShortcutStateManager = FLTShortcutStateManager()
+  ) {
+    self.channel = channel
+    self.shortcutStateManager = shortcutStateManager
+  }
+
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+    switch call.method {
+    case "setShortcutItems":
+      // `arguments` must be an array of dictionaries
+      let items = call.arguments as! [[String: Any]]
```

Comment:
Force unwrap it to keep the same behavior as ObjC. Whether to crash on platform code or to surface an error is discussed [here in this proposal](https://docs.google.com/document/d/1Iqt36mp_MWX7Ou2l9tQpUPjlNbh0rSRrni7X0DZK3pk/edit?resourcekey=0-XdZ6l6uykkhbBbivPl64lg#heading=h.1yb3rbn1zp2z).

Reply:
If wrong type it should crash: 

```
NSNumber *arr = @1;
for (NSDictionary *item in (NSArray *)arr) { // crash here
  NSLog(@"%@", item);
}
```
But I see your point about `nil` value. The equivalent swift should be this instead: 

```
if let arguments = call.arguments {
  // crash of wrong type
  items = arguments as! [[String:Any]]
} else {
  // no crash if nil
  items = []
}
```

Not sure if it's worth to do tho. The corresponding dart side guarantees that the argument is non-null. So the `else` block actually never run.

Reply:
SGTM